### ISSUE TITLE
Evaluating model checkpoint on validation set

### ISSUE THREAD

**Issue by DatHuynh** [openimages/dataset#67]
Have anyone tried to evaluate Resnet 101 image classification checkpoint (trained on V2 data) on the validation set? 
I got a weird result: only 26 MAP score which contradicts with this paper [Learning From Noisy Large-Scale DatasetsWith Minimal Supervision](https://arxiv.org/pdf/1701.01619.pdf) where they got 61 MAP on V1 dataset. 
Could it be because I resize the image to the maximum size of 256 when downloading them?

Thanks in advance.

**Comment by yoosan** [comment]
The evaluation metrics is not a standard mAP score. Maybe you shoule check the paper and find how they modify the mAP calculator.
